contributors/devel/api-conventions.md

```diff
@@ -317,14 +319,14 @@ following fields, but must contain at least `type` and `status` fields:
   // +optional
   LastHeartbeatTime  unversioned.Time  `json:"lastHeartbeatTime,omitempty" description:"last time we got an update on a given condition"`
```

Comment:
IIRC the whole concept of conditions was invented for NodeCondition and so some of the fields only make sense there (like this one).